src/Logikfabrik.Umbraco.Jet/Mappings/DefaultDataTypeDefinitionMapping.cs

```diff
@@ -118,39 +122,71 @@ private static Type GetNullableType(Type type)
         /// <summary>
         /// Gets the supported hints.
         /// </summary>
+        /// <param name="umbracoPropertyValueConvertersEnabled">The value of EnablePropertyValueConverters in umbracoSettings.config</param>
         /// <returns>The supported hints.</returns>
-        private static IDictionary<string, KeyValuePair<Type, DataTypeDefinition>> GetSupportedHints()
+        private static IDictionary<string, KeyValuePair<Type, DataTypeDefinition>> GetSupportedHints(bool umbracoPropertyValueConvertersEnabled)
```

Comment:
Sure, will do!

The old pickers have been labeled as obsolete and can only be used on sites that have been upgraded. 

They've also changed the data types of ContentPicker, CheckboxList, DropdownMultiple, MediaPicker, RelatedLinks, and Tags through the new property value converters. This setting lets you choose between the new and old types (true for new, false for old).